diff --git a/hakyll.cabal b/hakyll.cabal
index 927498c..84118ef 100644
--- a/hakyll.cabal
+++ b/hakyll.cabal
@@ -165,43 +165,43 @@ Library
 
   Build-Depends:
     base                 >= 4.8      && < 5,
-    binary               >= 0.5      && < 0.10,
-    blaze-html           >= 0.5      && < 0.10,
-    blaze-markup         >= 0.5.1    && < 0.9,
-    bytestring           >= 0.9      && < 0.11,
-    containers           >= 0.3      && < 0.7,
-    cryptohash           >= 0.7      && < 0.12,
-    data-default         >= 0.4      && < 0.8,
-    deepseq              >= 1.3      && < 1.5,
-    directory            >= 1.0      && < 1.4,
-    file-embed           >= 0.0.10.1 && < 0.0.12,
-    filepath             >= 1.0      && < 1.5,
-    lrucache             >= 1.1.1    && < 1.3,
-    mtl                  >= 1        && < 2.3,
-    network-uri          >= 2.6      && < 2.7,
-    optparse-applicative >= 0.12     && < 0.15,
-    parsec               >= 3.0      && < 3.2,
-    process              >= 1.6      && < 1.7,
-    random               >= 1.0      && < 1.2,
-    regex-tdfa           >= 1.1      && < 1.3,
-    resourcet            >= 1.1      && < 1.3,
-    scientific           >= 0.3.4    && < 0.4,
-    tagsoup              >= 0.13.1   && < 0.15,
-    template-haskell     >= 2.14     && < 2.15,
-    text                 >= 0.11     && < 1.3,
-    time                 >= 1.8      && < 1.10,
-    time-locale-compat   >= 0.1      && < 0.2,
-    unordered-containers >= 0.2      && < 0.3,
-    vector               >= 0.11     && < 0.13,
-    yaml                 >= 0.8.11   && < 0.12
+    binary,
+    blaze-html,
+    blaze-markup,
+    bytestring,
+    containers,
+    cryptohash,
+    data-default,
+    deepseq,
+    directory,
+    file-embed,
+    filepath,
+    lrucache,
+    mtl,
+    network-uri,
+    optparse-applicative,
+    parsec,
+    process,
+    random,
+    regex-tdfa,
+    resourcet,
+    scientific,
+    tagsoup,
+    template-haskell,
+    text,
+    time,
+    time-locale-compat,
+    unordered-containers,
+    vector,
+    yaml
 
   If flag(previewServer)
     Build-depends:
-      wai             >= 3.2   && < 3.3,
-      warp            >= 3.2   && < 3.3,
-      wai-app-static  >= 3.1   && < 3.2,
-      http-types      >= 0.9   && < 0.13,
-      fsnotify        >= 0.2   && < 0.4
+      wai,
+      warp,
+      wai-app-static,
+      http-types,
+      fsnotify
     Cpp-options:
       -DPREVIEW_SERVER
     Other-modules:
@@ -210,7 +210,7 @@ Library
 
   If flag(watchServer)
     Build-depends:
-      fsnotify        >= 0.2   && < 0.4
+      fsnotify
     Cpp-options:
       -DWATCH_SERVER
     Other-modules:
@@ -218,8 +218,8 @@ Library
 
   If flag(checkExternal)
     Build-depends:
-      http-conduit >= 2.2    && < 2.4,
-      http-types   >= 0.7    && < 0.13
+      http-conduit,
+      http-types
     Cpp-options:
       -DCHECK_EXTERNAL
 
@@ -231,8 +231,8 @@ Library
     Other-Modules:
       Hakyll.Web.Pandoc.Binary
     Build-Depends:
-      pandoc          >= 2.0.5    && < 2.8,
-      pandoc-citeproc >= 0.14     && < 0.17
+      pandoc,
+      pandoc-citeproc
     Cpp-options:
       -DUSE_PANDOC
 
@@ -244,38 +244,38 @@ Test-suite hakyll-tests
   Default-language: Haskell2010
 
   Other-modules:
-    Hakyll.Core.Dependencies.Tests
-    Hakyll.Core.Identifier.Tests
-    Hakyll.Core.Provider.Metadata.Tests
-    Hakyll.Core.Provider.Tests
-    Hakyll.Core.Routes.Tests
-    Hakyll.Core.Rules.Tests
-    Hakyll.Core.Runtime.Tests
-    Hakyll.Core.Store.Tests
-    Hakyll.Core.UnixFilter.Tests
-    Hakyll.Core.Util.String.Tests
-    Hakyll.Web.CompressCss.Tests
-    Hakyll.Web.Html.RelativizeUrls.Tests
-    Hakyll.Web.Html.Tests
-    Hakyll.Web.Tags.Tests
-    Hakyll.Web.Template.Context.Tests
-    Hakyll.Web.Template.Tests
+    Hakyll.Core.Dependencies.Tests,
+    Hakyll.Core.Identifier.Tests,
+    Hakyll.Core.Provider.Metadata.Tests,
+    Hakyll.Core.Provider.Tests,
+    Hakyll.Core.Routes.Tests,
+    Hakyll.Core.Rules.Tests,
+    Hakyll.Core.Runtime.Tests,
+    Hakyll.Core.Store.Tests,
+    Hakyll.Core.UnixFilter.Tests,
+    Hakyll.Core.Util.String.Tests,
+    Hakyll.Web.CompressCss.Tests,
+    Hakyll.Web.Html.RelativizeUrls.Tests,
+    Hakyll.Web.Html.Tests,
+    Hakyll.Web.Tags.Tests,
+    Hakyll.Web.Template.Context.Tests,
+    Hakyll.Web.Template.Tests,
     TestSuite.Util
 
   Build-Depends:
     hakyll,
-    QuickCheck                 >= 2.8  && < 2.14,
-    tasty                      >= 0.11 && < 1.3,
-    tasty-hunit                >= 0.9  && < 0.11,
-    tasty-quickcheck           >= 0.8  && < 0.11,
+    QuickCheck,
+    tasty,
+    tasty-hunit,
+    tasty-quickcheck,
     -- Copy pasted from hakyll dependencies:
-    base                 >= 4.8      && < 5,
-    bytestring           >= 0.9      && < 0.11,
-    containers           >= 0.3      && < 0.7,
-    filepath             >= 1.0      && < 1.5,
-    text                 >= 0.11     && < 1.3,
-    unordered-containers >= 0.2      && < 0.3,
-    yaml                 >= 0.8.11   && < 0.12
+    base,
+    bytestring,
+    containers,
+    filepath,
+    text,
+    unordered-containers,
+    yaml
 
   If flag(previewServer)
     Cpp-options:
@@ -306,9 +306,9 @@ Executable hakyll-init
 
   Build-depends:
     hakyll,
-    base      >= 4   && < 5,
-    directory >= 1.0 && < 1.4,
-    filepath  >= 1.0 && < 1.5
+    base,
+    directory,
+    filepath
 
 Executable hakyll-website
   Main-is:          site.hs
@@ -323,7 +323,7 @@ Executable hakyll-website
 
   Build-depends:
     hakyll,
-    base      >= 4     && < 5,
-    directory >= 1.0   && < 1.4,
-    filepath  >= 1.0   && < 1.5,
-    pandoc    >= 2.0.5 && < 2.8
+    base,
+    directory,
+    filepath,
+    pandoc
